src/crimson/osd/pg.h

```diff
@@ -362,6 +362,8 @@ class PG : public boost::intrusive_ref_counter<
   void check_blocklisted_watchers() final;
   void clear_primary_state() final {
     // Not needed yet
+
+    // projected_log = PGLog::IndexedLog();
```

Comment:
Maybe we need to uncomment this line?

Reply:
This is still WIP, I'll remove the DNM and ping you once it's ready

Reply:
Got it.